### PR TITLE
Added blank? method for string class

### PR DESCRIPTION
**Pull request by AlfonsoUceda**
lotus/controller#93

Added this method to use in this PR

### CODE REVIEW

**Comment by AlfonsoUceda** [comment]
lotus/controller#94

File: lib/lotus/utils/string.rb [hunk 1]
```diff
@@ -69,6 +69,33 @@ def initialize(string)
...
+      #   string = Lotus::Utils::String.new ''
+      #   string.blank? # => true
+      #
+      # @example non empty string
+      #   require 'lotus/utils/string'
+      #
+      #   string = Lotus::Utils::String.new 'lotus'
+      #   string.blank? # => false
+      def blank?
+        @string.nil? || @string.empty?
```
Comment (joneslee85) [inline lib/lotus/utils/string.rb]: the `nil` scenario is not possible because string is always casted to_s in constructor

**Comment by joneslee85** [comment]
To me, adding `blank?` to only String class does not add much benefits. What do you think about introducing `Lotus::Kernel.blank?(object)` instead? cc @jodosha

**Comment by joneslee85** [comment]
For the time being, I close this ticket.

**Pull request closed without merging**